Source/driver/Castro.cpp

```diff
@@ -454,13 +442,11 @@ Castro::read_params ()
       Radiation::read_static_params();
     }
 
-    // The CUDA MOL implementation doesn't currently do radiation.
-#ifdef AMREX_USE_CUDA
+    // radiation is only supported with CTU
     if (do_radiation && time_integration_method != CornerTransportUpwind) {
-        amrex::Error("Radiation is currently unsupported for MOL when using CUDA.");
+        amrex::Error("Radiation is currently unsupported for CTU time advancement.");
```

Comment:
I think this should say `Radiation is currently only supported for CTU time advancement`?

Reply:
fixed